R/topics.R

```diff
@@ -167,3 +167,82 @@ topic_must <- function(message, topic) {
     x = paste0("Not ", encodeString(topic, quote = "'"))
   ))
 }
+
+content_info <- function(content_entry, index, pkg, section) {
+
+  if (!grepl(".*::.*", content_entry)) {
+    topics <- pkg$topics[select_topics(content_entry, pkg$topics),]
+    tibble::tibble(
+      path = topics$file_out,
+      aliases = purrr::map2(topics$funs, topics$name, ~ if (length(.x) > 0) .x else .y),
+      name = list(topics$name),
+      title = topics$title,
+      icon = find_icons(topics$alias, path(pkg$src_path, "icons"))
+    )
+  } else { # topic from another package
+    names <- strsplit(content_entry, "::")[[1]]
+    pkg_name <- names[1]
+    topic <- names[2]
+    check_package_presence(pkg_name)
+
+    rd_href <- find_rd_href(sub("\\(\\)$", "", topic), pkg_name)
+    rd <- get_rd(rd_href, pkg_name)
+    rd_title <- extract_title(rd)
+    rd_aliases <- if (grepl("\\(\\)$", topic)) {
+      topic
```

Comment:
this is in the case where you want to highlight one function only, e.g. only `is_installed()` but not `check_installed()` from rlang. I am not entirely sure it's worth the added complexity as the title of the Rd is a description of all aliases in the Rd file anyway.

I suppose that for finer control, users should add docs themselves e.g. what bslib does for the font functions it exports from sass: https://github.com/rstudio/bslib/blob/112ae6e173cc80d4d47d4ef78e6086e47997d6cc/R/imports.R#L32

Reply:
Yeah, I think this is overthinking it a bit so you can remove it.